vmdb/app/presenters/menu/section.rb

```diff
@@ -8,6 +8,15 @@ def features
       Array(items).collect(&:feature).compact
     end
 
+    def visible?(userid)
+      store = Vmdb::PermissionStores.instance
+      tab   = Menu::Manager.tab_features_by_id(id)
+      auth  = store.can?(id) && User.current_user.role_allows_any?(:identifiers => tab)
```

Comment:
This line is the problem with the custom menus. Your yaml permission store does not capture RBAC features for the custom menu sections and items.